operations/app/src/modules/event_hub/main.tf

```diff
@@ -0,0 +1,53 @@
+terraform {
+    required_version = ">= 0.14"
+}
+
+resource "azurerm_eventhub_namespace" "eventhub_namespace" {
```

Comment:
Could we add a [`lifecycle.prevent_destroy = true`](https://www.terraform.io/docs/language/meta-arguments/lifecycle.html) to this module? This contains data that would be damaging if accidentally ran a terraform destroy on it.

Reply:
Added